linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_freq/rule.yml

```diff
@@ -19,6 +19,7 @@ identifiers:
 
 references:
     ospp: FAU_GEN.1
+    srg: SRG-OS-000051-GPOS-00024
```

Comment:
Nack. The control is for "The operating system must provide the capability to centrally review and analyze audit records from multiple components within the system."

Even if ``flush`` and ``freq`` are configured, RHEL would always have the capability to review and analyze udit records. Better rule to map these to could be "is audit utils installed?"

Reply:
If the value is not set, or is set to to a different number, audit records could be lost preventing the central review and analysis of the records.

Is there a better SRG to map the rule to?

Reply:
It is a requirement that the system admin have a basic understanding of how many events could be lost if the audit system is unable to log. Look at this as part of the loss of logging requirements.